src/client/Microsoft.Identity.Client/Platforms/Android/AndroidPlatformLogger.cs

```diff
@@ -9,6 +9,11 @@ namespace Microsoft.Identity.Client.Platforms.Android
 {
     internal class AndroidPlatformLogger : IPlatformLogger
     {
+        public void LogAlways(string message)
+        {
+            //NoOp
```

Comment:
Doesn't this defeat the purpose of "always log" ? You can use `Log.Info` here. 

aside: if you look at the methods available on Android `Log` object, one of the options is `wtf`, which is documented as "what a terrible failure" :D

Reply:
I wasn't sure which one to pick as there isnt a log always equivalent in android and I want sure how useful this info would be for PCA yet.
Isnt error below info so maybe we should use that instead?

Reply:
Yeah, I saw the wtf

Reply:
The information we log under this new category LogAlways is not really an error or a warning. So when we adapt our logger to the Android logger, we can use AndroidLogger.Info. The point here is that MSAL will always log these strings.